### ISSUE TITLE
Abbreviation pronounciation

### ISSUE THREAD

**Issue by vcjob** [fatchord/WaveRNN#131]
Hello everyone! 
Just want to find out if there is a way to force TTS pronounce a word letter by letter? Let's say, not '_api_', but '_eɪ-piː-aɪ_' instead? (btw, providing smth like _ei pi ai_, or _ai pi i_ and so on doesn't work well). 
Is it only possible when the training dataset has some phrases with letter pronounced one by one?
Any idea on how to make it work with a model, trained on lj dataset?
Thank you a lot!
(btw, if try to pass as an input phrase 'API' or some other short sentences, one will laugh a lot on the output. It is fixable?)
[__input_A P I_180k.zip](https://github.com/fatchord/WaveRNN/files/3497055/__input_A.P.I_180k.zip)

**Comment by vcjob** [comment]
> Hello everyone!
> Just want to find out if there is a way to force TTS pronounce a word letter by letter? Let's say, not '_api_', but '_eɪ-piː-aɪ_' instead? (btw, providing smth like _ei pi ai_, or _ai pi i_ and so on doesn't work well).
> Is it only possible when the training dataset has some phrases with letter pronounced one by one?
> Any idea on how to make it work with a model, trained on lj dataset?
> Thank you a lot!
> (btw, if try to pass as an input phrase 'API' or some other short sentences, one will laugh a lot on the output. It is fixable?)
> [__input_A P I_180k.zip](https://github.com/fatchord/WaveRNN/files/3497055/__input_A.P.I_180k.zip)

Let me reply myself. replacing letters with

**Comment by fatchord** [comment]
@vcjob I don't think that the repo in its current state will do a good job at pronouncing abbreviations. I think the best solution would be to implement phonemes. Not sure when I'll get around to that though.
